api/v2/endpoints/post-token-generate.md

```diff
@@ -121,7 +121,7 @@ Here is an example response when the policy respects user opt-out.
 }
 ```
 
-The [Client-Side Identity JavaScript SDK](../sdks/client-side-identity.md) uses this endpoint response payloads to establish and manage the user identity during a user session lifecycle.
+The [Client-Side JavaScript SDK (v2)](../sdks/client-side-identity.md) uses this endpoint response payloads to establish and manage the user identity during a user session lifecycle.
```

Comment:
why do we mention v2 here? presumably v3 and later versions will also do that

Reply:
It's there because it's part of the document title. I'd think v3 and later versions would have a separate doc and a different link, and we'd need to differentiate -- but I don't know. Kept it in for now. Let me know if you feel strongly about it and if so I will take it out.